shared/constants.js

```diff
@@ -4,7 +4,7 @@ const FUNCTIONS_API_URL = "https://api.scaleway.com/functions/v1beta1/regions";
 const CONTAINERS_API_URL =
   "https://api.scaleway.com/containers/v1beta1/regions";
 const REGISTRY_API_URL = "https://api.scaleway.com/registry/v1/regions";
-const ACCOUNT_API_URL = "https://api.scaleway.com/account/v2/projects";
+const ACCOUNT_API_URL = "https://api.scaleway.com/account/v3/projects";
```

Comment:
For reviewers: this change has **nothing** to do with this PR, but `v2` is deprecated, and was causing random `upstream request timeout` errors while creating projects for the tests (sometimes it takes slightly more than 30s, the default timeout of the API)... I've changed to the new v3 API (which is faster to create projects).